crypto/keys/internal/ecdsa/privkey.go

```diff
@@ -12,13 +12,7 @@ import (
 // p256Order returns the curve order for the secp256r1 curve
 // NOTE: this is specific to the secp256r1/P256 curve,
 // and not taken from the domain params for the key itself
-// (which would be a more generic approach for all EC)
-// In *here* we don't need to do it as a method on the key
-// since this code is only called for secp256r1
-// if called on a key:
-// func (sk PrivKey) pCurveOrder() *.big.Int {
-//     return sk.Curve.Params().N
-// }
```

Comment:
adding a second option in a comment (to use function instead of variable) is misleading. The second option was discussed in  the original PR and we opted for the variable. I don't think we need to bring that into the comment.

Reply:
I'm OK to remove it. It's a comment about a possible future way of doing this work.